src/components/BaseComponents/FormGroup/FormGroup.tsx

```diff
@@ -25,8 +26,17 @@ export default function FormGroup({
   testProps = {}
 }) {
 
+  const {errorMsgs} = useContext(ErrorMsgContext);
+  const [errMessage,setErrorMessage] = useState(errorText);
+  useEffect(()=>{
+    const found = errorMsgs.find((element) => (element.message.fieldId === id) || (element.message.fieldId.startsWith(name)));
+    if(!found){
+      setErrorMessage(errorText);
+    }
```

Comment:
this can be move to utils as this used in multiple file.

Reply:
Yeah done this .